api/views/course_admin.py

```diff
@@ -40,6 +43,28 @@ def register_user(self, request, pk=None):
         if course.is_registered(user):
             raise ValidationError(ERROR_MESSAGES.COURSE_REGISTRATION.ALREADY_REGISTERED)
 
-        # TODO: Implement the registration
+        # TODO: Implement the registration that uses of canvas_user_id which can be done from canvas api
 
         return Response(request.data)
+
+    @action(detail=True, methods=['post'], url_path='change-status')
+    def change_status(self, request, pk=None):
```

Comment:
This is not the way we want to implement it. It just a matter of project project convention there is nothing wrong with what you did. Please rename this to `update_registration` which gets 2 fields `is_verified` and `is_blocked` then it will update them. Then is the front-end you'll have only 1 function in the service for it and both button will you that service.